src/engine/ruleset/integrations/windows/documentation.yml

```diff
@@ -21,6 +21,14 @@ configuration: >
       </localfile>
   </ossec_config>
   ```
+  To detect events with sysmon, the following command must be executed:
+  ```
+  Sysmon64.exe -i sysmon_config.xml
+  ```
+  This configuration file can be configured by following this documentation
+  https://learn.microsoft.com/en-us/sysinternals/downloads/Sysmon
```

Comment:
Maybe:
The sysmon configuration attached in `agent/sysmon_config.xml` can be used to capture the events of this integration.  This configuration file can be configured by following this documentation
  https://learn.microsoft.com/en-us/sysinternals/downloads/Sysmon